Dockerfile

```diff
@@ -38,6 +40,11 @@ RUN ln -s ${ENTRYPOINT} ./entrypoint
 ENTRYPOINT ["./entrypoint"]
 
 
+FROM base AS no-build
+
+COPY --from=build-image /opt /opt
+
+
```

Comment:
Shouldn't this be above `build-stage`, even? Otherwise, the build-stage will fail for bad resolvers.

Reply:
Yes, you're right. I should pay attention to this and test with podman more frequently. Thanks!